mappings/net/minecraft/client/render/entity/model/VillagerResemblingModel.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS dfz net/minecraft/client/render/entity/model/VillagerResemblingModel
```

Comment:
This name is fine, just wondering if `VillagerLikeModel` might be simpler

Reply:
I think "resembling" is more accurate, though "like" is a correct term as well (it is used too often and slightly confusing)